README.md

```diff
@@ -43,21 +44,24 @@ You can start it:
 where /home/pi/projects/vzlogger-docker is the path to the directory containing the vzlogger.conf file and
 /dev/serial/by-id/usb-FTDI_FT230X_Basic_UART_D30A8U6N-if00-port0 is your device. You can pass several devices if you have them.
 
-Debian and Raspbian Packages
+Debian and Raspberry Pi OS Packages
 -------------
 
 [![Hosted By: Cloudsmith](https://img.shields.io/badge/OSS%20hosting%20by-cloudsmith-blue?logo=cloudsmith&style=flat-square)](https://cloudsmith.com)
 
-We now build Debian packages for amd64, armhf and arm64 and Raspbian packages 
-for armhf as part of our releases. Unfortunately Debian armhf packages do not 
-run on Raspberry Pi 1 although the architecture on the RPi is named armhf. 
-Using Raspian armhf packages fixes that.
+We now build Debian packages for amd64, armhf and arm64. Armel, which would be
+needed for Debian on RPi 1 hardware, is not supported.
 
-The ones attached to the release are meant for debian trixie. These and 
-packages for bookworm and bullseye are also provided through a repository 
-graciously provided by [Cloudsmith](https://cloudsmith.com).
+Raspberry Pi OS packages for armhf are also part of our releases. Unfortunately 
+Debian armhf packages do not run on Raspberry Pi 1 although the architecture 
+has been named armhf in Raspbian. Using "Raspbian armhf" packages fixes that.
```

Comment:
maybe provide a direct pointer to where those "Raspbian armhf" are to be found?
i.e. https://cloudsmith.io/~volkszaehler/repos/volkszaehler-org-project/packages/?q=distribution%3Araspbian+AND+architecture%3Aarmhf ?

or specific instructions on a sources.list entry for a raspberry pi 1?
according to https://cloudsmith.io/~volkszaehler/repos/volkszaehler-org-project/setup/#formats-deb :
`distro=raspbian arch=armel`?

Reply:
i think this sentence should be re-worded now that the above link already points to the raspbian packages.